api/rest/v1/bulk_observations_point_linked.md

```diff
@@ -0,0 +1,298 @@
+---
+layout: default
+title: Single Observation (linked)
+nav_order: 9
+parent: REST (v1)
+grand_parent: API
+published: false
+permalink: /api/rest/v1/bulk/observations/point/linked
+---
+
+# /v1/bulk/observations/point/linked
+
+Retrieve single [observations](/glossary.html#observation) of multiple [variables](/glossary.html#variable) at a set date for entities linked to a parent entity by the same property.
+
+This is useful for retrieving observations for all places within a parent place. For example, this could be getting the population of women in 2018 for all states in the United States.
+
+<div markdown="span" class="alert alert-info" role="alert" style="color:black; font-size: 0.8em">
+   <span class="material-icons md-16">info </span><b>Note:</b><br />
+   Currently, this endpoint only supports the `containedInPlace` property and `Place` entities. Support for other properties and entity types will be added in the future.
```

Comment:
I think we should list out all the valid <ancestor type, children type> for this API. We can talk about this more and may need to get this with BigQuery.

Reply:
+1, but leaving this as a todo for after we chat